videoframegrid.py

```diff
@@ -30,12 +30,13 @@ def _add_videoframe(self, videoframe):
         self.addWidget(videoframe, self.coordinates.x, self.coordinates.y)
         self.coordinates = self.coordinates.update_coordinates()
 
-    def _create_videoframe(self, vlc_instance, stream_options, quality):
+    def _create_videoframe(self, vlc_instance, stream_url, stream_options, quality):
         """Creates a new LiveVideoFrame object."""
-        return LiveVideoFrame(vlc_instance, stream_options, quality)
+        return LiveVideoFrame(vlc_instance, stream_url, stream_options, quality)
 
-    def add_new_videoframe(self, vlc_instance, stream_options, quality, move_handle):
+    def add_new_videoframe(self, parent, vlc_instance, stream_url, stream_options, quality):
         """Creates and adds a new LiveVideoFrame to the VideoFrameGrid."""
-        videoframe = self._create_videoframe(vlc_instance, stream_options, quality)
-        videoframe._move = move_handle
+        videoframe = self._create_videoframe(vlc_instance, stream_url, stream_options, quality)
```

Comment:
> I agree that we could change the stream_url, stream_options, quality to a dict as it were before though, since they "belong" together. Opinions?

Yes, agreed. Or even a class would encapsulate the properties nicely.